tests/test_nanopub_fetch.py

```diff
@@ -0,0 +1,24 @@
+from fairworkflows import Nanopub, FairData
+import pytest
+
+@pytest.mark.flaky(reruns=5)
```

Comment:
Cool, I didn't know about flaky tests

Reply:
I'd prefer to not have unit tests access external resources, but I understand that you want to test this. Maybe we can do something about this later, like mocking the nanopub server, and keep this as a separate test outside of the regular unit tests? Just some ideas